tests/integrationtest/l4_crd_test.go

```diff
@@ -1476,6 +1476,256 @@ func TestCreateDeleteL4RuleSSLDefaultValues(t *testing.T) {
 	ResetMiddleware()
 }
 
+func TestL4RuleSSLCustomValuesLicenseCloudServices(t *testing.T) {
+	g := gomega.NewGomegaWithT(t)
+
+	// setting license to enterprise
+	SetupLicense := func(license string) {
```

Comment:
Can you please shift this to lib so that it can be reused ?

Reply:
Have move it to lib.go